### DISCUSSION TITLE
onkokb module failed because not in store

### DISCUSSION THREAD

**Discussion by lmanchon** [KarchinLab/open-cravat#246] [Usability & Feature Requests]
oc module install onkokb
No modules to install found

oc module info onkokb
NOT IN STORE
NOT INSTALLED

How to install onkokb module ?

**Comment by RachelKarchin** [comment]
The command should work but there is a "c" in oncokb

Try this: oc module install oncokb

**Comment by lmanchon** [comment]
sorry i have misspelled the word, i have installed oncokb with oc module install oncokb, iinstallation works fine,
but in annotation the corresponding column of oncokb still empty.

**Comment by kmoad** [comment]
Oncokb needs an api token provisioned by the user. OpenCRAVAT cannot re-distribute the token. There are more details on the oncokb module panel in the store. 

In brief, get a token from https://api.oncokb.org/oncokb-website/api. Then put it in a subfolder of the oc modules directory. To find the location of your modules directory, run `oc config md`. Then put the token in to `<md>/annotators/oncokb/data/token.txt`

Let us know if this works.

**Comment by lmanchon** [comment]
it seems token isn't free for all users.

**Comment by kmoad** [comment]
Yes, I believe it's free for academic users only. Sorry that we couldn't be more help here.
